src/services/admin/admin-router.ts

```diff
@@ -0,0 +1,48 @@
+import { Router } from "express";
+import { StatusCodes } from "http-status-codes";
+import { Database } from "../../database";
+import RoleChecker from "../../middleware/role-checker";
+import { Role } from "../auth/auth-models";
+import dotenv from "dotenv";
+import { validateQrHash } from "./admin-utils";
+
+dotenv.config();
+
+const adminRouter = Router();
+
+adminRouter.post(
+    "/scan/",
```

Comment:
minor question for @AydanPirani: do we want this to end in a '/' or not? bc some endpoints have the trailing slash and others don't

Reply:
ideally not, but it's fine